### PR TITLE
[Task 23] BlankNavActivity javadocs

### PR DESCRIPTION
**Pull request by candidcalcifer**

### CODE REVIEW

**Review by A-Bunch-Of-Atoms — DISMISSED** [review]

File: app/src/main/java/com/courseproject/tindar/BlankNavActivity.java [hunk 1]
```diff
@@ -21,6 +21,10 @@
 public class BlankNavActivity extends AppCompatActivity {
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/BlankNavActivity.java:21]: needs JavaDoc for the class

File: app/src/main/java/com/courseproject/tindar/BlankNavActivity.java [hunk 2]
```diff
@@ -21,6 +21,10 @@
...
     private AppBarConfiguration mAppBarConfiguration;
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/BlankNavActivity.java:22]: needs JavaDoc for the class attribute

File: app/src/main/java/com/courseproject/tindar/BlankNavActivity.java [hunk 3]
```diff
@@ -21,6 +21,10 @@
...
 
+    /**
+     * Creates the home page and navigation drawer that persists when user is logged in.
+     * Associated with each of the fragments that are accessible from the navigation drawer.
+     */
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/BlankNavActivity.java:27]: needs @param

File: app/src/main/java/com/courseproject/tindar/BlankNavActivity.java [hunk 4]
```diff
@@ -52,13 +56,22 @@ protected void onCreate(Bundle savedInstanceState) {
         NavigationUI.setupWithNavController(navigationView, navController);
     }
 
+    /**
+     * Populates menu with items
+     */
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/BlankNavActivity.java:61]: needs @param

**Review by jenniferjinyoungyang — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/courseproject/tindar/BlankNavActivity.java [hunk 5]
```diff
@@ -52,13 +62,24 @@ protected void onCreate(Bundle savedInstanceState) {
         NavigationUI.setupWithNavController(navigationView, navController);
     }
 
+    /**
+     * Populates menu with items
+     *
+     * @param menu collection of menu items associated with the activity
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/BlankNavActivity.java:68]: can you add @return here?

**Review by jenniferjinyoungyang — APPROVED** [review]

**Review by yiwxng — APPROVED** [review]

**Review by kresimirgotovac — APPROVED** [review]

**Review by kresimirgotovac — APPROVED** [review]

**Pull request merged**